app/Http/Requests/Admin/UserRequest.php

```diff
@@ -39,7 +39,7 @@ public function rules()
                 'required',
```

Comment:
check xem các điều kiện đủ chưa nhé

Reply:
I don't know how to fix regex with "^". I tried to search on Google but imposible.